subsys/fs/fuse_fs_access.c

```diff
@@ -176,9 +176,15 @@ static int fuse_fs_access_readdir(const char *path, void *buf,
 		 * directory but FUSE strips the trailing slashes from
 		 * directory names so add it back.
 		 */
-		char mount_path[PATH_MAX];
+		char mount_path[PATH_MAX] = {0};
```

Comment:
Note for the future: if you are planning to overwrite the buffer with `whatevercpy` do not initialize it this way, because this will be replaced by loop putting 0 in the array; just do all the copy you need ant place the final, as you will know where to place it anyway.